mix.exs

```diff
@@ -0,0 +1,62 @@
+defmodule CsGuide.Mixfile do
+  use Mix.Project
+
+  def project do
+    [
+      app: :cs_guide,
+      version: "0.0.1",
+      elixir: "~> 1.4",
```

Comment:
does not match version we're testing with in `travis.yml` consider updating in next iteration.